### DISCUSSION TITLE
[Announcement] Homework 0-3 is graded

### DISCUSSION THREAD

**Discussion by Hhro** [prosyslab-classroom/cs424-program-reasoning#351] [Announcements]
Hi all,
Homework 0-3 is graded.

We made a dummy homework named "Homework 0-3" at KLMS Week4 and uploaded your score there.
This score will be applied as a multiplication ratio to the final one.
For example, if your final score is 90, and the score of this is 3, you would get 92.7.

Thanks.

**Comment by Lee-Janggun** [comment]
Thank you for the quick grading! May I know what the score is out of? It says that the average is 0.64, so can I take that it is out of 1.00?

**Comment by Hhro** [comment]
No, scores are graded between 0 and 3.

**Comment by KihongHeo** [comment]
Hi,

The score of each submission is one of 0, 1, 2, or 3 based on the following criteria:
- Top-down
- Understanding
- Critical thinking

Since this is a bonus homework, it is a bit strictly scored.

Thanks for all your submission. I enjoyed reading the articles.

**Comment by Lee-Janggun** [comment]
Thank you for the clarification!
